content/docs/start/experiments/experiment-collaboration.md

```diff
@@ -46,10 +46,17 @@ $ dvc exp pull origin "soupy-leak"
 
 <tab title="VSCode Extension">
 
-You can right-click on the experiment row and select the `Share to Studio`
-action:
+You can right-click on the experiment row and select the `Push Selected` action:
+
+![VSCode Push](/img/vscode-sharing.gif)
+
+</tab>
+
+<tab title="Studio">
+
+Manage shared experiments from [Studio]:
```

Comment:
Not a blocker but feels like potentially misleading as the tabs are usually alternatives but in this case you would need to use one of the other 2 tabs to see what is shown here

Reply:
Would you suggest dropping it? It doesn't quite fit, but I think it's still useful to show what pushing looks like in Studio.

Reply:
Let's come back to it if needed. Going to merge to unblock this one.